### ISSUE TITLE
Support NestJS 9 & Typorm 0.3.x

### ISSUE THREAD

**Issue by mtdewulf** [ringcentral/nestjs-json-api#43]
TypeORM 0.3.x has some improvements, and unfortunately [some breaking changes](https://github.com/typeorm/typeorm/blob/master/CHANGELOG.md#030-2022-03-17).  The latest version of `nestjs/typeorm` now [supports (requires) TypeORM  0.3.x](https://github.com/nestjs/typeorm/pull/1233).  Bottom line: if you want to use NestJS 9, you need to use TypeORM 0.3.x.

I've [made a fork](https://github.com/ringcentral/nestjs-json-api/compare/master...arrowsup:nestjs-json-api:master) that addresses the above issue.  However, there is one main problem with it at this point: [typeorm-seeding does not yet support 0.3.x](https://github.com/w3tecch/typeorm-seeding/issues/228), and so I've had to remove the seed loading command from this branch.

Bottom line, this isn't a big change, but it isn't backward compatible -- and not all dependencies to this repo have been upgraded, yet.  I'm wondering, **what are the maintainers' plans for handling this upgrade?**

**Comment by klerick** [comment]
Hi, 
We have plan to upgrade and try to do it in the nearest future. We have project with NestJs 8, Typeorm 0.2.x and Angular 13 (We using [sdk](https://www.npmjs.com/package/json-api-nestjs-sdk) for client too). Our deadline is in August and then we’ll start upgrading to the new version. During upgrade we will try to fix and refactor problems that we found while using the lib

**Comment by backnight** [comment]
Hello guys,

Any update on this?

Thanks a lot!

**Comment by klerick** [comment]
I am working on new version. But I think it will be version with breaking changes.

**Comment by backnight** [comment]
I'm fine with that.

On my side I'm trying to launch a new project based on NestJS 9 and using this library to make a quick JSON API compatible endpoint but I'm currently stuck.

@mtdewulf I tried to use the fork you did but without success. Did it worked for you?

**Comment by mtdewulf** [comment]
@backnight, I ended up abandoning the fork.  For my project, I went a different direction.

**Comment by alifey** [comment]
@mtdewulf I'm in the same situation as @backnight, can you give some more details?

**Comment by mtdewulf** [comment]
@alifey, I was working on a NestJS / TypeORM backend for a [React Admin Data Provider](https://marmelab.com/react-admin/DataProviders.html).  Using a [JSON API Data Provider](https://github.com/henvo/ra-jsonapi-client) was one option to solve that, but I ended up implementing the backend for their [Simple REST Data Provider](https://github.com/marmelab/react-admin/tree/master/packages/ra-data-simple-rest).

**Comment by alifey** [comment]
@mtdewulf, thanks for the explanation.

**Comment by klerick** [comment]
@alifey @mtdewulf I have publish new version with support typeorm 3.x.x and NestJS 9.x.x
new version is https://www.npmjs.com/package/json-api-nestjs/v/3.0.0-beta.1
I do not know what will happen whit it repo. so I have forked https://github.com/klerick/nestjs-json-api
